src/lib/util/ModUtil.php

```diff
@@ -296,6 +306,9 @@ public static function setVar($modname, $name, $value = '')
 
     /**
      * The setVars method sets multiple module variables.
+     * @deprecated at Core-1.4.1
+     * @see \Zikula\ExtensionsModule\Api\VariableApi::setAll()
```

Comment:
this is the proper use of `@see`

http://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.see.pkg.html